features/booking_admin.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Admin booking page
+  As an administrator
+  In order to sett up roles for booking a facilities
+  I would like to be able to manage the facilities booking settings.
+
+  Scenario: Create booking schedule for a facility
+    Given there is a facility named "Laundry Room"
+    And I am on the facility show page
```

Comment:
yes, its because i we want to test the that there is a link on the show page to the facility